filament/src/details/Camera.cpp

```diff
@@ -96,11 +96,11 @@ void UTILS_NOINLINE FCamera::setCustomProjection(mat4 const& p,
 
 void UTILS_NOINLINE FCamera::setCustomEyeProjection(math::mat4 const* projection, size_t count,
         math::mat4 const& projectionForCulling, double near, double far) {
-    ASSERT_PRECONDITION(count >= CONFIG_STEREOSCOPIC_EYES,
+    const Engine::Config& config = mEngine.getConfig();
+    ASSERT_PRECONDITION(count >= config.stereoscopicEyeCount,
             "All eye projections must be supplied together, count must be >= "
-            "CONFIG_STEREOSCOPIC_EYES(%d)",
-            CONFIG_STEREOSCOPIC_EYES);
-    for (uint8_t i = 0; i < CONFIG_STEREOSCOPIC_EYES; i++) {
+            "config.stereoscopicEyeCount (%d)", config.stereoscopicEyeCount);
```

Comment:
I think `count` should be either exactly `1` or `config.stereoscopicEyeCount` (and also documented), because it's an `Engine` wide setting and with `OVR_multiview` shaders won't be able to do less.

Reply:
Why 1? This API method requires all eyes to be set at the same time, so I think we can require it to be exactly `config.stereoscopicEyeCount`.

Reply:
okay